core/src/main/scala/tamer/config.scala

```diff
@@ -59,11 +55,7 @@ object KafkaConfig {
     env("KAFKA_TRANSACTIONAL_ID").as[String]
   ).mapN(KafkaConfig.apply)
 
-  final val fromEnvironment: ZLayer[Blocking with Clock, TamerError, Has[KafkaConfig]] =
-    ZIO
-      .runtime[Clock with Blocking]
-      .flatMap(implicit runtime => kafkaConfigValue.load[Task])
-      .refineToOrDie[ConfigException]
-      .mapError(ce => TamerError(ce.error.redacted.show, ce))
-      .toLayer
+  final val fromEnvironment: Layer[TamerError, KafkaConfig] = ZLayer {
+    kafkaConfigValue.load[Task].refineToOrDie[ConfigException].mapError(ce => TamerError(ce.error.redacted.show, ce))
```

Comment:
we never explored `zio-config`, did we? especially since we dropped refined. Using `zio-config`, we won't need the interop, right?

Reply:
We won't need it in the core, yes. Happy for us to explore zio-config though